### PR TITLE
ENH: Hide background segment after Grow from seeds effect

### PR DESCRIPTION
**Pull request by cpinter**
Determine whether a segment is background by checking corner voxel values (if 5 or more are non-zero, then background)

### CODE REVIEW

**Comment by cpinter** [comment]
@lassoan We talked about this change. What do you think?

File: Modules/Loadable/Segmentations/EditorEffects/Python/AbstractScriptedSegmentEditorAutoCompleteEffect.py [hunk 1]
```diff
@@ -49,6 +49,22 @@ def __del__(self, scriptedEffect):
...
+  def isBackgroundLabelmap(labelmapOrientedImageData):
+    if labelmapOrientedImageData is None:
+      return False
+    # If five or more corner voxels of the image contain non-zero, then it is background
+    extent = labelmapOrientedImageData.GetExtent()
+    numberOfFilledCorners = 0
+    for i in [0,1]:
+      for j in [2,3]:
+        for k in [4,5]:
+          if labelmapOrientedImageData.GetScalarComponentAsFloat(extent[i],extent[j],extent[k],0) > 0:
```
Comment (lassoan) [inline Modules/Loadable/Segmentations/EditorEffects/Python/AbstractScriptedSegmentEditorAutoCompleteEffect.py]: Add a check that the extent is not empty (extent[i]<=extent[i+1] along all three axes).

**Comment by jcfr** [comment]
👍 After addressing @lassoan  comments

**Comment by cpinter** [comment]
Integrated in https://github.com/Slicer/Slicer/commit/08890700b0692ef079eec2c09ff84b3fc7cb8c49

**Pull request closed without merging**